setup.py

```diff
@@ -1,3 +1,21 @@
+"""
```

Comment:
For .py files you should comment using only `"""` (i.e. remove the `/*`, `*` and `*/`)